.rubocop.ruby3-rails8.yml

```diff
@@ -0,0 +1,76 @@
+---
+# shared configuration of rubocop
+
+require:
+  - rubocop-rails
+  - rubocop-performance
+  - rubocop-minitest
+
+# Shared config for all projects regarding all cops
+AllCops:
+  DisplayCopNames: true
+  Exclude:
+    - bin/**/*
+    - exe/**/*
+    - tmp/**/*
+    - public/**/*
+    - vendor/**/*
+    - node_modules/**/*
+    - db/schema.rb
+    # Evaluate git config to exclude all ignored files and folders from rubocop checks.
+    # The indentation is to not show the error "Invalid child element in a block sequence" in the editor (by yaml check).
+    # Since this file is pre-processed by ERB, the indentation is not visible in the final file.
+    # See: https://docs.rubocop.org/rubocop/configuration.html#pre-processing
+      <% `git status --ignored --porcelain`.lines.grep(/^!! /).each do |path| %>
+    - <%= path.sub(/^!! /, '').sub(/\/$/, '/**/*') %>
+      <% end %>
+  NewCops: enable
+  TargetRailsVersion: 8.0
+  TargetRubyVersion: 3.0
```

Comment:
Do we want to include a more recent Ruby version?
Apparently is version 3.0 not receiving any further updates https://www.ruby-lang.org/en/downloads/branches/

Reply:
Thanks for catching this! I previously thought RuboCop only supported pinning to major Ruby versions (e.g., 3.0.0), but that's outdated.

RuboCop is actually quite smart about version detection:
- For Ruby: It checks *.gemspec, .ruby-version, .tool-versions, and Gemfile.lock (in that order)
- For Rails: It checks Gemfile.lock or gems.locked, defaulting to Rails 5.0 if neither exists

Since both Ruby and Rails versions are auto-detected, we can remove the explicit version settings entirely.

This makes handling the shared config easier by **a lot**!
I will modify this PR to take the latest findings into account.

Thank you for bringing this to my attention!

Reply:
I removed the version pins from the config. The old config will stay for compatibility reasons until we use the new utils gem in all repos.